_i18n/en.yml

```diff
@@ -109,7 +109,7 @@ download:
 buyfiro:
   heading: Buy Firo
   heading2: One of the ways to get Firo is to acquire it on one of the exchanges. Below you will find a list of exchanges and coin-swapping services that support Firo.
-  predisclaimer: Disclaimer
+  predisclaimer: Disclaimer&#58;&nbsp;

```

Comment:
I put the colon and space here because in some languages like Chinese and Japanese the colon is different from the colon in English